plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-products-controller.php

```diff
@@ -235,6 +263,15 @@ protected function prepare_objects_query( $request ) {
 			$args['meta_key'] = $ordering_args['meta_key']; // WPCS: slow query ok.
 		}
 
+		/*
+		 * When the suggested products ids is not empty,
+		 * filter the query to return only the suggested products,
+		 * overwriting the post__in parameter.
+		 */
+		if ( ! empty( $this->suggested_products_ids ) ) {
+			$args['post__in'] = $this->suggested_products_ids;
+		}
```

Comment:
Sorry for not noticing this earlier, but this can also be removed right?

Reply:
Don't sorry at all. It's my mistake. Thanks for reviewing. Addressed.